frontend/src/locale/index.ts

```diff
@@ -354,7 +354,18 @@ export const messages = {
     "grants.form.fields.email": "Email",
     "grants.form.fields.email.description":
       "Here you will receive a notification about your application.",
-    "grants.form.fields.age": "Age",
+    "grants.form.fields.ageGroup": "Age",
+    "grants.form.fields.ageGroup.values.selectOption": "Select Option",
+    "grants.form.fields.ageGroup.values.range_less_than_10":
```

Comment:
is anyone ever going to select this :P 😂

Reply:
Yes, at PyCon 11 we had one grant request for a 9 years old.